flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/canal/CanalJsonDeserializationSchema.java

```diff
@@ -105,7 +106,8 @@ private CanalJsonDeserializationSchema(
                         false, // ignoreParseErrors already contains the functionality of
                         // failOnMissingField
                         ignoreParseErrors,
-                        timestampFormat);
+                        timestampFormat,
+                        allowUnescapedControlChars);
```

Comment:
Please use fields to store the value of the `allowUnescapedControlChars` and `timestampFormat `.